tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,332 @@
+# global field power for each sensor type of the two :class:`~mne.Evoked`
+# objects:
 
-##############################################################################
-# Save the source time courses to disk:
+aud_evoked = aud_epochs.average()
+vis_evoked = vis_epochs.average()
 
-stc.save('mne_dSPM_inverse')
+mne.viz.plot_compare_evokeds(dict(auditory=aud_evoked, visual=vis_evoked),
+                             show_legend='upper left',
+                             show_sensors='upper right')
 
-##############################################################################
-# Now, let's compute dSPM on a raw file within a label:
+###############################################################################
+# We can also get a more detailed view of each :class:`~mne.Evoked` object
+# using other plotting methods such as :meth:`~mne.Evoked.plot_joint` or
+# :meth:`~mne.Evoked.plot_topomap`. Here we'll examine just the EEG channels,
+# and see the classic auditory evoked N100-P200 pattern over dorso-frontal
+# electrodes, then plot scalp topographies at some additional arbitrary times:
 
-fname_label = data_path + '/MEG/sample/labels/Aud-lh.label'
-label = mne.read_label(fname_label)
+# sphinx_gallery_thumbnail_number = 13
+aud_evoked.plot_joint(picks='eeg')
+aud_evoked.plot_topomap(times=[0., 0.08, 0.1, 0.12, 0.2], ch_type='eeg')
 
 ##############################################################################
-# Compute inverse solution during the first 15s:
+# Evoked objects can also be combined to show contrasts between conditions,
+# using the :func:`mne.combine_evoked` function. A simple difference can be
+# generated by negating one of the :class:`~mne.Evoked` objects passed into the
+# function. We'll then plot the difference wave at each sensor using
+# :meth:`~mne.Evoked.plot_topo`:
 
-from mne.minimum_norm import apply_inverse_raw  # noqa
-start, stop = raw.time_as_index([0, 15])  # read the first 15s of data
-stc = apply_inverse_raw(raw, inverse_operator, lambda2, method, label,
-                        start, stop)
+evoked_diff = mne.combine_evoked([aud_evoked, -vis_evoked], weights='equal')
+evoked_diff.pick_types('mag').plot_topo(color='r', legend=False)
 
 ##############################################################################
-# Save result in stc files:
+# Time-frequency analysis
+# ^^^^^^^^^^^^^^^^^^^^^^^
+#
+# The :mod:`mne.time_frequency` submodule provides implementations of several
+# algorithms to compute time-frequency representations, power spectral density,
+# and cross-spectral density. Here, for example, we'll compute for the
+# auditory epochs the induced power at different frequencies and times, using
+# Morlet wavelets:
 
-stc.save('mne_dSPM_raw_inverse_Aud')
+frequencies = np.arange(7, 30, 3)
+power = mne.time_frequency.tfr_morlet(aud_epochs, n_cycles=2, return_itc=False,
+                                      freqs=frequencies, decim=3)
```

Comment:
Shouldn't this go after the epochs section?

Also, perhaps we could add a list of examples at the end of each section for people interested in learning more (e.g., like in [scikit-learn](https://scikit-learn.org/stable/modules/svm.html))

Reply:
I would add a note here that this just shows the evoked response and that this is for illustration purposes. I would maybe point to an example using somato dataset for which it makes more sense.